.pre-commit-config.yaml

```diff
@@ -65,7 +65,6 @@ repos:
                   RedfishOSCredentials-16faa37e-4b6a-4891-9028-242de65a3b70)|
               plugins/uefi-capsule/tests/test\.bmp|
               src/tests/auth/firmware\.xml\.gz|
-              src/tests/history_v[12]\.db|
```

Comment:
well; we could pretty easily generate those at build time with a sqlite `custom_target()` and a `.sqlite` "source" file, no?

Reply:
True.  that's probably the better way to do with the history db.
The first commit OK?

Reply:
Yup, of course - thanks.